tests/libYARP_math/MathTest.cpp

```diff
@@ -635,10 +635,23 @@ class MathTest : public UnitTest {
         report(0, "checking Quaternion class");
         Quaternion q1;
         Quaternion q2;
+        Quaternion q3;
+        Quaternion q4;
         Matrix m;
         Matrix m_check;
         Vector v_check;
 
+        double vA[3] = {20.0, 30.0, 45.0};
+        double vB[3] = {12.0, 60.0, 25.0};
```

Comment:
It is not blocking (60 rad = 3437.7 deg is still a valid angle, even if a strange one : ) ).